### PR TITLE
[14.0][MIG] stock_reserve_rule

### PR DESCRIPTION
**Pull request by sebalix**

### CODE REVIEW

**Review by mmequignon — APPROVED** [review]

File: stock_reserve_rule/migrations/13.0.1.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
```
Comment (simahawk) [inline stock_reserve_rule/migrations/13.0.1.1.0/post-migration.py]: you should drop this

File: stock_reserve_rule/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
```
Comment (simahawk) [inline stock_reserve_rule/readme/CONTRIBUTORS.rst:1]: you?

File: stock_reserve_rule/tests/test_reserve_rule.py [hunk 3]
```diff
@@ -0,0 +1,651 @@
+# Copyright 2019 Camptocamp (https://www.camptocamp.com)
+
+from odoo import exceptions, fields
+from odoo.tests import common
+
+
+class TestReserveRule(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```
Comment (simahawk) [inline stock_reserve_rule/tests/test_reserve_rule.py:10]:
```suggestion
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

**Review by simahawk — CHANGES_REQUESTED** [review]

File: stock_reserve_rule/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (sebalix) [reply stock_reserve_rule/readme/CONTRIBUTORS.rst:1]: Don't know, didn't do so much to migrate it :)

**Review by simahawk — APPROVED** [review]

**Pull request merged**